shared/js/textChatView.jsx

```diff
@@ -199,10 +199,13 @@ loop.shared.views.chat = (function(mozL10n) {
         "text-chat-entries": true
       });
 
+      var hideHelloWelcome = loop.shared.utils.isDesktop() ||
+        this.props.messageList.length > 0;
```

Comment:
This doesn't feel quite right, as the welcome to will go away once a message has been typed or a context tile displayed.

Maybe we should search the message list, or kinda give up and get the room name from the active room store.